src/site/content/en/blog/sign-in-form-best-practices/index.md

```diff
@@ -0,0 +1,540 @@
+---
+title: Sign-in form best practices
```

Comment:
@samdutton I changed the URL from `signin-form-best-practice` to `sign-in-form-best-practices`